[STTSPG-507, STTSPG-1106] Update Backfilled Notices documentation for Status Pages (#36749)
DataDog/documentation: content/en/incident_response/status_pages/_index.md

You can publish updates over time and mark the notice as **Resolved** when the issue is fully mitigated.

> #### Backfill a degradation

Reviewer: Maybe documentation will have input, but what should heading size be? Since it is #### it kind of gets lost in the sidebar nav, but I understand why you put that since it is a type of degradation. 

Maybe it should be same level as "[Publish a degradation](https://docs-staging.datadoghq.com/julia.benais/update-backfilled-notices-doc/incident_response/status_pages/#publish-a-degradation)", since in the UI, it is a separate button/branch

Backfilled degradations allow you to retroactively document service disruptions that were not previously announced. Each update can be assigned its original timestamp, so the incident timeline appears accurately in your uptime history.

> From a status page, under **Publish Notice**, click **Publish Backfilled Notice**, select **Degradation**, then provide:

Reviewer 2: ```suggestion
From a status page, select the dropdown next to **Publish Notice**, click **Publish Backfilled Notice**, select **Degradation**, then provide:
```
Documentation team may have some guidance on exact wording here

> From a status page, select the dropdown next to **Publish Notice**, click **Publish Backfilled Notice**, select **Degradation**, then provide:

Reviewer 3: ```suggestion
From a status page, select the dropdown next to **Publish Notice**, select **Publish Backfilled Notice** > **Degradation**, then provide:
```

| Field | Description |
| ---- | ---- |
| **Notice title** | Short, clear description of the incident <br>*Example: Increased error rates in US region* |

> | **Updates** | One or more timestamped updates reconstructing the incident timeline. Each update includes a started at timestamp, status (Investigating, Identified, Monitoring, or Resolved), description, and impacted components. |

Reviewer 2: In the UI, we gate the ability to post a fully fledged degradation and only allow Investigating -> Resolved.
Might be worth to differentiate that in the UI vs API that allows many updates to be filled for a backfilled degradation

You can post updates if plans change or reschedule the maintenance window as needed.

### Backfill a maintenance window

Backfilled maintenance windows allow you to retroactively document planned downtime that was not previously announced. Each update can be assigned its original timestamp, so the maintenance timeline appears accurately in your uptime history.

> From a status page, select the dropdown next to **Publish Notice**, click **Publish Backfilled Notice**, select **Scheduled Maintenance**, then provide:

Reviewer 3: ```suggestion
From a status page, select the dropdown next to **Publish Notice**, select **Publish Backfilled Notice** > **Scheduled Maintenance**, then provide:
```
